### PR TITLE
feat: Add roles, activated to JWT access token

### PR DESCRIPTION
**Pull request by howar31**
# Issue
Add roles, activated to JWT access token

# Notice

# Dependency

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205522078452851

### CODE REVIEW

File: controllers/account.go [hunk 1]
```diff
@@ -298,7 +298,22 @@ func (mc *MembershipController) TokenDispatch(c *gin.Context) (int, gin.H, error
 		return http.StatusInternalServerError, gin.H{"status": "error", "message": "cannot get user data"}, err
 	}
 
-	accessToken, err = utils.RetrieveV2AccessToken(user.ID, user.Email.ValueOrZero(), acccessTokenExpiration)
+	roles := make([]map[string]interface{}, len(user.Roles))
+	for i, role := range user.Roles {
```
Comment (Aylie-Chou) [inline controllers/account.go:302]: Question: if the user has multiple roles, what role would we supposed to return here ?

Comment (howar31) [reply controllers/account.go:302]: In current implementation, all roles will be returned in an array (same as API response).  For multiple roles support, it should be implemented in another commit imo.

Comment (Aylie-Chou) [reply controllers/account.go:302]: As the disscusion of [this thread](https://github.com/twreporter/cronjob/pull/104#discussion_r1287880344), would we apply it in this pr ? (or we could apply the new multiple roles logic when conflict fixing in this sprint)

Comment (howar31) [reply controllers/account.go:313]: done.  both token and API are fixed.

**Review by Aylie-Chou — APPROVED** [review]
feel free to discuss if you have any problem while resolving conflict

**Comment by howar31** [comment]
I've resolved the conflict by:
1. order by `roles`.`weight` DESC first
2. order by `users_roles`.`updated_at` DESC

**Review by Aylie-Chou — APPROVED** [review]
LGTM, we could update other getting roles function with the order `roles.weight desc, users_roles.updated_at desc` in following PR.

**Pull request merged**